mne/io/constants.py

```diff
@@ -809,3 +809,6 @@ def __setattr__(self, attr, val):
 # MNE epochs bookkeeping
 FIFF.FIFFB_MNE_EPOCHS_SELECTION    = 3800  # the epochs selection
 FIFF.FIFFB_MNE_EPOCHS_DROP_LOG     = 3801  # the drop log
+
+# MNE annotations
+FIFF.FIFFB_MNE_ANNOTATIONS          = 3810  # annotations
```

Comment:
don't forget to also update https://github.com/mne-tools/mne-fiff-constants, preferably with a PR so there is some public activity showing it